### PR TITLE
#1827 - Change dmn id and requirements

### PR DESCRIPTION
**Pull request by andrepestana-aot**
- Changed dmn id and requirements for dmnPartTimeAssessmentDecisions and dmnFullTimeAssessmentDecisions.
image

### CODE REVIEW

File: sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-decisions.dmn [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
+<definitions xmlns="https://www.omg.org/spec/DMN/20191111/MODEL/" xmlns:dmndi="https://www.omg.org/spec/DMN/20191111/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" xmlns:biodi="http://bpmn.io/schema/dmn/biodi/2.0" xmlns:camunda="http://camunda.org/schema/1.0/dmn" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="dmnFullTimeAssessmentDecisions" name="dmnFullTimeAssessmentDecisions" namespace="http://camunda.org/schema/1.0/dmn" exporter="Camunda Modeler" exporterVersion="4.11.1" camunda:diagramRelationId="1a575ead-1e6d-468c-901a-766a35569b38" modeler:executionPlatform="Camunda Cloud" modeler:executionPlatformVersion="8.1.0">
```
Comment (ann-aot) [inline sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-decisions.dmn:2]: maybe your modeler needs an update, I see a lower version

File: sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-decisions.dmn [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (andrewsignori-aot) [reply sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-decisions.dmn:1]: With two physical DMNs(DRD) files grouping many DMNs. The recouceName is related to the physical files while the others are the individual DMNs. Does it answer the question?
image

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍

File: sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-decisions.dmn (hunk 1 above)
Comment (andrepestana-aot) [reply sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-decisions.dmn]: updated. Thanks!

**Review by ann-aot — APPROVED** [review]
LGTM @andrepestana-aot 👍

**Review by guru-aot — APPROVED** [review]
LGTM

**Review by dheepak-aot — APPROVED** [review]
Looks Good

**Pull request merged**